group_vars/aap/aap_config_controller_credential.yml

```diff
@@ -0,0 +1,53 @@
+---
+controller_credentials:
+  - credential_type: Machine
+    name: "Machine Credential"
+    description: "SSH Machine Connection with IdM credentials"
+    organization: "{{ aap_config_organization }}"
+    inputs:
+      username: "{{ default_ssh_vm_user }}"
+      password: "{{ techuser_ansible_global_password }}"
```

Comment:
Shall we rename this var to more generic name?

Reply:
leave it for now